helm/prometheus-operator/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: '0.4.0'
+description: Prometheus Operator
+name: prometheus-operator
+version: 0.0.1
```

Comment:
@jwtodd Is this related to nats at all?  should it be in this PR?

Reply:
yeah ... i'd like to get grafana/surveyor back on the table. testing/verifying locally atm

Reply:
Maybe a new PR for that then?  Seems like this isn't a blocker for NATS and the event layer.

Reply:
or is it specifically for monitoring NATS message flow?